core-services/prow/02_config/netobserv/network-observability-operator/_prowconfig.yaml

```diff
@@ -1,3 +1,9 @@
+branch-protection:
+  orgs:
+    netobserv:
+      repos:
+        network-observability-operator:
+          unmanaged: true
```

Comment:
any reason why we're removing prow to manage our protected branches? If I am understanding this correct, the the tests jobs are triggered on PRs looking to merge to main will no longer have tests as `Required`.

Reply:
I used to have our upstream unit tests as required, which I found better because they're faster to run and less susceptible to environmental issues. Not only prow's branch protection added more flakiness in required tests but also removed the previously defined required tests.
I prefer to manage them from upstream/github - and then we can have the discussion of what to set required or not.